app/forms/webauthn_setup_form.rb

```diff
@@ -64,11 +65,13 @@ def consume_parameters(params)
   def name_is_unique
     return unless WebauthnConfiguration.exists?(user_id: @user.id, name: @name)
     if @platform_authenticator
-      errors.add :name, I18n.t('errors.webauthn_platform_setup.unique_name'), type: :unique_name
+      while WebauthnConfiguration.exists?(user_id: @user.id, name: @name)
+        @name = "#{@name} (#{++1})"
```

Comment:
This incrementer would only go up to 2. What happens if there's already a `(2)` suffixed name?

Reply:
I see. I have included an incrementer that gets added to during the 'while' loop.